cantools/database/can/formats/sym.py

```diff
@@ -921,7 +921,10 @@ def _dump_message(message: Message, signals: List[Signal], min_frame_id: TypingO
         if multiplexer_signal.byte_order == 'big_endian':
             m_flag = '-m'
         hex_multiplexer_id = format(multiplexer_id, 'x').upper()
-        message_str += f'Mux="{hex_multiplexer_id}" {_convert_start(multiplexer_signal.start, multiplexer_signal.byte_order)},{multiplexer_signal.length} {hex_multiplexer_id}h {m_flag}\n'
+        multiplexer_signal_name = hex_multiplexer_id
+        if multiplexer_signal.name != '':
+            multiplexer_signal_name = multiplexer_signal.name
```

Comment:
flipping around the logic here is probably easier to read:
```suggestion
        multiplexer_signal_name = multiplexer_signal.name
        if not multiplexer_signal_name:
            multiplexer_signal_name = hex_multiplexer_id
```